UnitsNet.Tests/CustomCode/SpeedTests.cs

```diff
@@ -68,6 +68,28 @@ public class SpeedTests : SpeedTestsBase
 
         protected override double MillimetersPerHourInOneMeterPerSecond => 3600000;
 
+        protected override double FeetPerHourInOneMeterPerSecond => 11811.023622;
+
+        protected override double FeetPerMinuteInOneMeterPerSecond => 196.850393701;
+
+        protected override double InchesPerHourInOneMeterPerSecond => 141732.283465;
+
+        protected override double InchesPerMinuteInOneMeterPerSecond => 2362.20472441;
+
+        protected override double InchesPerSecondInOneMeterPerSecond => 39.3700787402;
+
+        protected override double YardsPerHourInOneMeterPerSecond => 3937.00787402;
+
+        protected override double YardsPerMinuteInOneMeterPerSecond => 65.6167979003;
+
+        protected override double YardsPerSecondInOneMeterPerSecond => 1.093613298338;
+
+        protected override double UsSurveyFeetPerSecondInOneMeterPerSecond => 3.280833333333;
+
+        protected override double UsSurveyFeetPerMinuteInOneMeterPerSecond => 196.85;
+
+        protected override double UsSurveyFeetPerHourInOneMeterPerSecond => 11811;
+
```

Comment:
We prefer using scientific notation in tests. For example, 1.1811E4 instead of 11811.

Rest is fine, I found no errors in quantity definitions and test values. Good job 🥇

Reply:
Done! :)

Reply:
Thanks, but I guess you forgot 10 other tests?

Reply:
Oh sorry about that. I'm looking at the other tests and can't seem to figure out the convention. (e.g. RotationalSpeedTests, PressureTests)

Reply:
No you misunderstood. You have added 11 new units for Speed quantity, so you wrote 11 tests, one for each. 

In your last commit you changed the test of `UsSurveyFeetPerHourInOneMeterPerSecond` into scientific notation.

I meant that you forgot to change the other tests for the units which you added, which includes Foot, UsSurveyFoot, Yard and Inch units.

Did I express myself clearly?

Reply:
Well it's not critical, let's not wait for this, we can return and change them into scientific notation later on. I'm merging this pull request.